docs/source/qvm.rst

```diff
@@ -111,8 +110,7 @@ with very high probability, get different results each time.
 The ``.wavefunction(...)`` method
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The QVM is a virtual machine. As such, we can directly inspect the wavefunction of
-a program, even without measurements, using the ``.wavefunction(...)`` method:
+We can directly inspect the wavefunction of a program, even without measurements, using the ``.wavefunction(...)`` method:
```

Comment:
isn't this not true?

Reply:
Yeah, I feel like I made comments on this last week? @ryankarle The vast majority of this file is incorrect. I can help plug in today and tomorrow to re write it.

Reply:
Ah, I didn't commit some of the fixes. Will keep working on it